### ISSUE TITLE
Unexpected warning in hpp file.

### ISSUE THREAD

**Issue by upizpp** [jbenden/vscode-c-cpp-flylint#198]
As far as I know, files with the extension of *hpp* should also belong to the type of **Header file**, rather than **source code** files with the extension of *cpp*.
 Therefore, I think using '# pragma once' in HPP files is allowed, but this extension gave me a warning.
> #pragma once in main file [-Wpragma-once-outside-header]

**Comment by gregersn** [comment]
I have my include files in a separate folder, and added 

```
  "c-cpp-flylint.excludeFromWorkspacePaths": [
    "include"
  ],
```

as a workaround, to avoid cppcheck getting passed the include files.

Not sure if it is ideal.

**Comment by dsgler** [comment]
same problem. I turn off it by set
"c-cpp-flylint.clang.extraArgs": [
        "-Wno-pragma-once-outside-header"
    ],

hope this bug could be fixed
